tests/sphinx/test_github_links.py

```diff
@@ -0,0 +1,191 @@
+import pathlib
+import sys
+
+from django.test import SimpleTestCase
+
+
+class GitHubLinkTests(SimpleTestCase):
+    @classmethod
+    def setUpClass(cls):
+        # The file implementing the code under test cannot be imported through
+        # standard means, so include its parent in the pythonpath for the
+        # duration of the tests.
+        cls.ext_path = str((pathlib.Path(__file__).parents[2] / "docs/_ext").resolve())
+        sys.path.insert(0, cls.ext_path)
```

Comment:
Should we use a couple of `addClassCleanup` here instead of defining `tearDownClass`?